rules/msk_app_consume_groups_test.go

```diff
@@ -0,0 +1,93 @@
+package rules
+
+import (
+	"testing"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/stretchr/testify/require"
+	"github.com/terraform-linters/tflint-plugin-sdk/helper"
+)
+
+func Test_MSKAppConsumeGroupsRule(t *testing.T) {
+	rule := &MSKAppConsumeGroupsRule{}
+
+	for _, tc := range []struct {
+		name     string
+		files    map[string]string
+		expected helper.Issues
+	}{
+		{
+			name: "single bad entry",
+			files: map[string]string{
+				"file.tf": `
+module "my-app" {
+	consume_groups = ["my-bad-group"]
+}
+`,
+			},
+			expected: []*helper.Issue{
+				{
+					Rule:    rule,
+					Message: "'consume_groups' must contain only '.' separated values, but 'my-bad-group' is not such a value",
```

Comment:
I was thinking of a message more like "consumer groups must be prefixed with the name of the team using it."
Don't think we should put something in the message about the separator. I think documenting it in the examples should be enough

Reply:
> I was thinking of a message more like "consumer groups must be prefixed with the name of the team using it." Don't think we should put something in the message about the separator. I think documenting it in the examples should be enough

:+1: [b32e67f](https://github.com/utilitywarehouse/tflint-ruleset-kafka-config/pull/35/commits/b32e67f0ef57fbb13755062584dd0ba75959b04d)